frontend/views/components/Modal/ModalMixins.js

```diff
@@ -13,38 +12,17 @@ const modaMixins = {
     ModalClose
   },
   mounted () {
-    sbp('okTurtles.events/on', CLOSE_MODAL, this.hide)
-    keyboardEvent = window.addEventListener('keyup', (e) => {
-      if (e.key === 'Escape') {
-        this.close()
-      }
-    })
+    sbp('okTurtles.events/on', CLOSE_MODAL, this.close)
   },
   beforeDestroy () {
-    sbp('okTurtles.events/emit', UNLOAD_MODAL)
-    window.removeEventListener('keyup', keyboardEvent)
+    sbp('okTurtles.events/off', CLOSE_MODAL)
   },
   methods: {
     close (e) {
-      sbp('okTurtles.events/emit', CLOSE_MODAL)
-    },
-    hide () {
-      // Only the last open modal should close
-      // TODO: find a more eleguant way to identify the current instance
-      // Maybe saving the name in the modal ?
-      const instance = this.$parent.$options._componentTag
-      const query = this.$route.query
-      if (query) {
-        const subcontent = query.subcontent
-        if (subcontent) {
-          if (subcontent === instance) this.isActive = false
-          // Don't do anything if it's not the last modal
-          return false
-        }
-      }
-      // Remove event only if it's the last modal
-      sbp('okTurtles.events/off', CLOSE_MODAL)
       this.isActive = false
+      setTimeout(() => {
+        sbp('okTurtles.events/emit', UNLOAD_MODAL)
+      }, 300)
```

Comment:
I suppose this 300ms is to "wait" for the modal animation to finish, right? Maybe you can leave it commented, it might not be straightforward for other developers.